packages/core/src/components/DataItem/index.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import withStyles, { WithStylesProps } from '../../composers/withStyles';
+
+import Text from '../Text';
+
+function DataItem({
+  label,
+  link,
+  floatLinkRight,
+  inline,
+  children,
+  cx,
+  styles,
+}: {
+  label: string;
+  link?: NonNullable<React.ReactNode>;
+  floatLinkRight?: boolean;
+  inline?: boolean;
+  children: NonNullable<React.ReactNode>;
+} & WithStylesProps) {
+  return (
+    <div className={cx(inline && styles.data_item_inline)}>
```

Comment:
generally the `_` is used as a separator between the base style and the modifier (so, for example `styles.link, floatRight && styles.link_floatRight`). stuff after that is camelCase again. since there is no base style data item and just an inline modifier, i'd just name the style `inline` and do `inline && styles.inline`

Reply:
Oh got it, made the change.

I've been adding in base class names so that when you go to look at Elements, you can see `dataItem_inline` instead of just `inline`, but I guess that's usually not a concern?